Remove mention of the GPS plugin (#7966)
qgis/QGIS-Documentation: docs/user_manual/introduction/qgis_configuration.rst

.. index:: GPS
.. _gps_options:

GPS settings
------------

.. _defining_new_device:

GPSBabel
........

`GPSBabel <https://www.gpsbabel.org/>`_ converts waypoints, tracks, and routes between popular GPS receivers 
such as Garmin or Magellan and mapping programs like Google Earth or Basecamp. 
Literally hundreds of GPS receivers and programs are supported.
QGIS relies on GPSBabel to interact with these devices
and :ref:`manipulate their data <gps_algorithms>`.

#. First you have to define the :guilabel:`Path to GPSBabel` binaries.
#. Then you may want to add your device.
   You can update devices list using |symbologyAdd| :sup:`Add new device`
   or |symbologyRemove| :sup:`Remove device` button.
#. For each device:

   * you provide a :guilabel:`Device name`
   * you configure different :guilabel:`Commands` QGIS will use while interacting with it,
     such as:

     * :guilabel:`Waypoint download` from the device
     * :guilabel:`Waypoint upload` to the device
     * :guilabel:`Route download` from the device
     * :guilabel:`Route upload` to the device
     * :guilabel:`Track download` from the device
     * :guilabel:`Track upload` to the device

>      This will probably be a GPSBabel command,
>      but you can use any other command line program that can create a GPX file.

Reviewer: ```suggestion
     While the commands are usually GPSBabel commands, you can also use any other command line program that can create a GPX file.
```
